tests/sbiutils_test.py

```diff
@@ -417,7 +417,7 @@ def test_z_scoring_structured():
     t = np.arange(0, 1, 0.001)
     x_sin = np.sin(t * 2 * torch.pi * 5)
     x = np.vstack([[(x_sin * (i + 1)) + (i * 2)] for i in range(10)])
-    t_batch = torch.tensor(x)
+    t_batch = torch.FloatTensor(x)
```

Comment:
quick fix for now, we will refactor this test in a later PR.

Reply:
Yes, i thought so too, but didn't have time to dig in further.